centreon/src/Core/TimePeriod/Domain/Model/TimeRange.php

```diff
@@ -79,6 +82,8 @@ public function getRanges(): array
      */
     private function isValidTimeRangeFormat(string $timeRange): bool
     {
+        Assertion::minLength($timeRange, 11, 'TimeRange::timeRange');
```

Comment:
By moving the assertion, we hide the fact that the isValidTimeRangeFormat method can raise an exception
```suggestion
```